sdk/src/main/java/com/hedera/hashgraph/sdk/FeeComponents.java

```diff
@@ -0,0 +1,155 @@
+package com.hedera.hashgraph.sdk;
+
+import com.google.protobuf.InvalidProtocolBufferException;
+
+
+
+public class FeeComponents {
+    
+    
+    private long min; // A minimum, the calculated fee must be greater than this value
```

Comment:
Please add the comments as doc comments above each field instead of in line comments. This applies to all inline comments.
```java
/**
 * A minimum, the calculated fee must be greater than this value
 * /
```